liferay-echo/liferay-portal
# ISSD-1406 Migrate market module AUI main.js file to purely javascript

Steps to Verify
========
1. Go to Applications Menu -> Control Panel -> Marketplace.
2. Check that Store and Purchased portlets are rendered correctly and navigate through them.
